### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by pcothenet**
I've been using this fork in production for 1 year+, since the original `winston-rollbar` is MIA. Fixed a few bugs and updated the dependencies. 

Would be great if we could merge this so I can switch back to get my dependency from npm instead of GitHub :)

### CODE REVIEW

File: lib/winston-rollbar.js [hunk 1]
```diff
@@ -70,9 +70,12 @@ Rollbar.prototype.log = function (level, msg, meta, callback) {
         }
     }
 
-    if (['warn','error'].indexOf(level) > -1 && (msg instanceof Error || meta instanceof Error)) {
+    if (['warn','error','critical'].indexOf(level) > -1 && (msg instanceof Error || meta instanceof Error)) {
         var error;
-         meta.level = meta.level || level;
+        meta.level = meta.level || level;
+        if (meta.level === 'warn') {
+            meta.level = 'warning';
```
Comment (pcothenet) [inline lib/winston-rollbar.js]: Because Rollbar only works with the level 'warning', not `warn`.

File: lib/winston-rollbar.js [hunk 2]
```diff
@@ -70,9 +70,12 @@ Rollbar.prototype.log = function (level, msg, meta, callback) {
         }
     }
 
-    if (['warn','error'].indexOf(level) > -1 && (msg instanceof Error || meta instanceof Error)) {
+    if (['warn','error','critical'].indexOf(level) > -1 && (msg instanceof Error || meta instanceof Error)) {
```
Comment (pcothenet) [inline lib/winston-rollbar.js]: to also send `critical` errors to rollbar

Comment (em0ney) [inline lib/winston-rollbar.js]: winston doesn't have a critical level by default.  If using syslog levels, it is 'crit', but this module doesn't support syslog levels anyway... yet :)

File: lib/winston-rollbar.js (hunk 1 above)
Comment (em0ney) [inline lib/winston-rollbar.js]: Yeah, good one!

**Comment by em0ney** [comment]
hi @pcothenet - thanks for your PR updates.  Can you please fix the merge conflicts?

**Pull request closed without merging**